<!--<?php
// $Log: onsa.php,v $
// Revision 1.4  2002/06/11 22:00:08  mavcunha
// Links para o Clinical e para o Ludwig foram consertados.
//
// Revision 1.3  2002/02/21 15:34:07  mavcunha
// space and link problems
//
// Revision 1.2  2002/02/20 13:48:05  mavcunha
// Finished texts and logos, all ok except for the lack of CITRI description
//
// Revision 1.1  2002/02/19 20:50:24  mavcunha
// Added onsa page, with ONSA projects and links.
//
// $Name:  $
// $Author: mavcunha $
?> -->
<?php require_once("includes/main.inc"); ?>
<!-- BEGIN localfile -->
<p class="newsboxtitle">ONSA Projects</p>

<center>
<br>
<a href="http://watson.fapesp.br/onsa/Genoma3.htm"><img src="/bit/images/logo_onsa.jpg" alt="ONSA Network Logo" border="0" align="middle"></a>
<br>
</center>


<table cellpadding="2" cellspacing="2" border="0" width="660">
   <tr>
   	<td class="boxtitle" valign="middle" colspan="2">Human</td>
  </tr>
   <tr>
      <td class="boxtitleblue" valign="top" width="50%">Human Cancer Genome Project</td>
      <td class="boxtitleblue" valign="top" width="50%">Transcript Finishing Initiative</td>
   </tr>
   <tr>
   	<td width="50%">
	<a href="http://www.ludwig.org.br/">
      	<img  src="/bit/images/logo_hcgp.png" alt="HCGP Logo" border="0" align="left">
	</a>
	<p style="text-align: justify;">FAPESP/LICR-HCGP is a program for human gene discovery and human coding region compilation based on a novel concept for the high throughput sequencing of human open reading frames. It constitutes part of the second phase of the ONSA initiative that builds on the success of the X. fastidiosa project . It is the first venture by the ONSA program into the highly competitive area of human genome sequencing. It also represents the first of the ONSA projects that will be an equal partnership between FAPESP and a private research organization, the Ludwig Institute for Cancer Research.</p>
	</td>
   	<td width="50%">
	<a href="http://www.compbionet.org.br/transcript/">
      	<img  src="/bit/images/logo_tfi.png" alt="TFI Logo" border="0" align="left">
	</a>
	<p style="text-align: justify;" >The aim of this project is to validate the structure and to generate sequences for 4,000 full-length human genes in a two-year period. As opposed to conventional full-length cDNA sequencing projects such as MGC, gene structure will be predicted and validated without the construction of highly elaborated cDNA libraries. Gene prediction will rely mainly on genomic sequences available trough the Human Genome Project. Genomic sequences will be used as a scaffold for EST (conventional and ORESTES) mapping as well as for gene prediction by computational analysis. Primers for validation of putative genes will be generated based on EST mapping, computational exon prediction and patterns of genomic organization, such as average gene size, intron/exons size and distribution. Validation will be done mainly by RT-PCR, insert subcloning and sequencing. As a final achievement, a databank for gene cataloguing will be created and also will provide additional information as chromosomal location, expression pattern, alternative transcripts forms etc...</p>
	</td>
   </tr>
   <tr>
      <td class="boxtitleblue" valign="top" width="50%">Clinical Genomics</td>
      <td valign="top" width="50%"></td>
   </tr>
   <tr>
   	<td width="50%">
	<a href="http://ctc.fmrp.usp.br/clinicalgenomics/appSP/">
      	<img  src="/bit/images/logo_clinical.png" alt="Clinical Genomics Logo" border="0" align="left">
	</a>
	<p style="text-align: justify;"> Clinical Genomics can be grossly defined as the use of information obtained from DNA sequencing, gene discovery or patterns of gene expression for medical purposes. This has been an important target of the FAPESP/Ludwig Human Cancer Genome Project since its launching, although the first two years of the project was dedicated mostly to obtaining sequence data of expressed genes from cancer cells. Now FAPESP and the Ludwig Institute for Cancer Research propose to expand the project objectives to include an application of clinical genomics, i. e., the analysis of gene expression in neoplastic cells in order to identify genes that are relevant for cancer diagnosis, prognosis, definition of subtypes, prediction of response to different therapeutic approaches or that may represent targets for drug development.</p>
	</td>
   	<td width="50%"></td>
   </tr>
   <tr>
   	<td class="boxtitle" valign="middle" colspan="2">Microbial</td>
  </tr>
   <tr>
      <td class="boxtitleblue" valign="top" width="50%"><i>Xylella fastidiosa</i></td>
      <td class="boxtitleblue" valign="top" width="50%"><i>Xanthomonas citri</i></td>
   </tr>
   <tr>
   	<td width="50%">
	<a href="http://aeg.lbi.ic.unicamp.br/xf/">
      	<img  src="/bit/images/logo_xylella.png" alt="Xylella Logo" border="0" align="left">
	</a>
	<p style="text-align: justify;">Xylella fastidiosa is a fastidious, xylem-limited bacterium, which causes a range of economically important plant diseases. This website contains information about the complete genome sequence of X. fastidiosa clone 9a5c, shown to cause citrus variegated chlorosis, a serious disease of orange trees. Citrus Variegated Chlorosis (CVC), which was first recorded in Brazil in 1987, affects all commercial sweet orange varieties. Symptoms include conspicuous variegations on older leaves, with chlorotic areas on the upper side and corresponding light brown lesions, with gum-like material on the lower side. Affected fruits are small, hardened, and of no commercial value. A strain of Xylella fastidiosa was first identified as the causal bacterium in 1993 and found to be transmitted by sharpshooter leafhoppers in 1996. CVC control is at present limited to removing infected shoots by pruning, the application of insecticides and the use of healthy plants for new orchards. In addition to CVC, other strains of X. fastidiosa cause a range of economically important plant diseases including Pierce's disease of grapevine, alfalfa dwarf, phony peach disease, periwinkle wilt, and leaf scorch of plum, and are also associated with diseases in mulberry, pear, almond, elm, sycamore, oak, maple, pecan and coffee. The triply-cloned X. fastidiosa 9a5c was derived from the pathogenic culture 8.1b obtained in 1992 in Bordeaux (France) from CVC-affected Valencia sweet orange twigs collected in Macaubal (Sao Paulo, Brazil) on May 21, 1992. Strain 9a5c produces typical CVC symptoms on inoculation into experimental citrus plants, as well as into Nicotiana tabacum and Catharantus roseus, two novel experimental hosts.</p>
	</td>
   	<td width="50%">
	<a href="http://genoma4.iq.usp.br/xanthomonas/">
      	<img  src="/bit/images/logo_xantho.png" alt="Xanthomonas Logo" border="0" align="left">
	</a>
	<p style="text-align: justify;">No description available</p>
	</td>
   </tr>
   <tr>
      <td class="boxtitleblue" valign="top" width="50%">Functional Genomics</td>
      <td class="boxtitleblue" valign="top" width="50%">Agronomical & Environmental Genomes</td>
   </tr>
   <tr>
   	<td width="50%">
	<a href="http://watson.fapesp.br/funcional/main.htm">
      	<img  src="/bit/images/logo_funcional.png" alt="Funcional Logo" border="0" align="left">
	</a>
	<p style="text-align: justify;">A bactéria Xylella fastidiosa foi escolhida como alvo do primeiro projeto do Programa Genoma da FAPESP pela sua relevância como patógeno responsável pela Clorose Variegada do Citros (CVC). Para que a informação sobre seu genoma, cujo sequenciamento vem progredindo rapidamente, seja explorada de forma a gerar potenciais benefícios para nossa agricultura, é essencial que seja iniciado imediatamente um programa de análise das funções biológicas dos genes identificados.
	</p>
	</td>
   	<td width="50%">
	<a href="http://watson.fapesp.br/AEG/agro.htm" >
      	<img  src="/bit/images/logo_aeg.png" alt="Agronomical and Environmental Logo" border="0" align="left">
	</a>
	<p style="text-align: justify;">Given the many ONSA laboratories specialized in organisms related to agronomy and the environment and that various sequencing projects were proposed by some of them, the time is ripe for launching a coordinated multi-genome sequencing effort in this area.                                                         Participants of this project are expected to be involved in various sequencing sub-projects all at once, during two years, beginning with the genomes of the two bacterial plant pathogens Xylella fastidiosa of grapevines and Leifsonia xyli  subsp. xyli of sugarcane. While these two genomes are being sequenced, participants of the AEG group are expected to propose and consolidate other sequencing projects. These will be processed by the Genome Coordination Committee and evaluated by outside referees. The proponent of a given project is expected to coordinate it in all relevant scientific and technical details. </p>
	</td>
   </tr>
   <tr>
   	<td class="boxtitle" valign="middle" colspan="2">Plant</td>
  </tr>
   <tr>
      <td class="boxtitleblue" valign="top" width="50%">Sugar Cane EST Genome Project</td>
      <td valign="top" width="50%"></td>
   </tr>
   <tr>
   	<td width="50%">
	<a href="http://sucest.lad.dcc.unicamp.br/en/">
      	<img  src="/bit/images/logo_sucest.png" alt="Sugar Cane  Logo" border="0" align="left">
	</a>
	<p style="text-align: justify;">The main goal of SUCEST is to undertake a large scale EST program by sequencing random clones from cDNA libraries prepared from several sugarcane tissues (calli, root, stalk, etiolated leaves, flowers, developing seed, etc). The aim of the project is to identify around 50,000 sugarcane genes. The project will be considered finished when this goal is reached or when 300,000 reads are deposited. Computational analysis and similarity searches against known sequences deposited in the databases will be performed in order to putatively identify the ESTs. The information provided by SUCEST can be exploited by the research community in studies aimed to use the sugarcane genes as source of markers for agriculturally significant characteristics and also to provide molecular basis for studies of plant growth and development that could be further used to solve questions in plant physiology, biochemistry, cell biology, pathology and ultimately plant breeding. The cDNA clones whose nucleotide sequence has been determined will be used to complement the sugarcane molecular map and fabricate microarrays of immobilized DNAs which will be used to survey expression of each gene in different sugarcane tissues under different environmental conditions.</p>
	</td>
   	<td width="50%"></td>
   </tr>
   <tr>
   	<td class="boxtitle" valign="middle" colspan="2">Parasite</td>
  </tr>
   <tr>
      <td class="boxtitleblue" valign="top" width="50%"><i>Schistosoma mansoni</i></td>
      <td valign="top" width="50%"></td>
   </tr>
   <tr>
   	<td width="50%">
	<a href="http://verjo18.iq.usp.br/schisto/">
      	<img  src="/bit/images/logo_shisto.png" alt="Shistosima Logo" border="0" align="left">
	</a>
	<p style="text-align: justify;">The project started on April, 2001. The aim of this project is to sequence in the next 18 months 120,000 EST clones generated by the ORESTES low-stringency RT-PCR amplification technique (Dias-Neto et al., PNAS U.S.A. 97: 3491-3496, 2000) using S. mansoni mRNA. We expect to generate 40 to 50 million bases of S. mansoni cDNA, which might represent on average an estimated 1 to 1.5 fold coverage of the parasite transcriptome. Different stages in the life cycle of the parasite will be studied. We expect to collect approximately 15 to 20 thousand ESTs from each different stage of the cycle including cercariae, miracidium, adult worms (male/female), eggs and 7-day circulating schistosomula.</p>
	</td>
   	<td width="50%"></td>
   </tr>
   
</table>
<!-- END localfile --> 
<?php $t->pparse("out","Footer"); ?>
